### PR TITLE
Alternative lib

### PR DESCRIPTION
**Pull request by Daniel2894**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,6 +33,8 @@
     "mochawesome": "^3.0.2"
   },
   "dependencies": {
+    "es6-promise": "^4.2.4",
```
Comment (sorregom) [inline package.json]: why did you add this dependency?

**Review by sorregom — CHANGES_REQUESTED** [review]
please check comments

File: package.json (hunk 1 above)
Comment (Daniel2894) [reply package.json]: It was a mistake, already removed.

**Review by sorregom — APPROVED** [review]
LGTM
